spec/index.bs

```diff
@@ -121,6 +121,8 @@ A data model extension consists of the following documents:
 
 ## Extension Schema File ## {#extension-definition}
 
+Issue: Decide on the schema definition language. I.e. is JSON Schema the right pick for us here?
```

Comment:
Should we also propose what are other options we may consider if not JSON Schema? or This is suppose to be an open question?